example-projects/actix-web-example/src/main.rs

```diff
@@ -39,3 +39,33 @@ async fn main() -> std::io::Result<()> {
     .run()
     .await
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[actix_rt::test]
+    async fn test_post() {
+        use actix_web::{test, App};
+        use cloudevents::AttributesReader;
+
+        let expected = v10::minimal();
```

Comment:
Does `cloudevents::Event::default()` make more sense here?

Reply:
Yes in this case. This test was just a showcase of using test fixtures from the external project. It is not a very good test 🙂